announcement/announcement-api/api/src/bundle/announcement_ca.properties

```diff
@@ -196,6 +196,8 @@ revise.required.star=Els elements obligatoris estan marcats amb un <span class\=
 revise.update=Actualitzeu el formulari i trieu el bot\u00f3 corresponent al final.
 revise.announce=Av\u00eds
 revise.announcement.body=Cos
+revise.highlight = <strong>Destacar aquest anunci.</strong> Ser\u00e0 m\u00e9s prominent a la llista d\u00b4 anuncis.
```

Comment:
Please use two strings and do not put markup in translation strings, markup must be in the VM file.

I believe the translation should be
"Ser\u00e0 m\u00e9s prominent a la llista d\u0027anuncis."

Reply:
Ok. I will apply it

Reply:
This has been marked as resolved but I don't see any change @JuanDavid102